app/controllers/application_controller.rb

```diff
@@ -11,6 +11,10 @@ def current_member
     current_user.member
   end
 
+  helper_method def current_user?
```

Comment:
Devise has a [user_signed_in?](https://github.com/heartcombo/devise#controller-filters-and-helpers) helper that provides similar functionality.